vignettes/cookbook_running_the_analysis.Rmd

```diff
@@ -229,13 +229,13 @@ In practice, if a company has a sector code with `borderline` is `TRUE` and *is*
 
 If you can obtain sector codes in any one of those classification systems for your loan book, it is strongly recommended using them. If you do not have your loans mapped to any of these systems, you can either run the matching without a sector classification (see above), or you can use a custom sector classification system.
 
-If you want to use your own custom sector classification system, you will need to create a file that follows the same structure as the sector classification files in `r2dii.data::sector_classifications`, that is, the file needs to contain the following columns (of the following types):
+If you want to use your own custom sector classification system, you will need to create a file that follows the same structure as the sector classification files in `sector_classifications`, that is, the file needs to contain the following columns (of the following types):
 
 ```{r cols_custom_sector_classification, echo = FALSE, results = 'asis'}
-cat(paste0("- `", names(r2dii.data::sector_classifications), "`: ", lapply(r2dii.data::sector_classifications, class),collapse = "\n"))
+cat(paste0("- `", names(sector_classifications), "`: ", lapply(sector_classifications, class),collapse = "\n"))
 ```
 
-Please notice that when manually preparing a custom sector classification file, you need to decide for every code of your mapping system which PACTA sector the code maps to. This requires the system to be relatively granular to capture the nuances of the supply chains of the companies in your loan book. As discussed above, any sector code that is not unambiguously in scope or out of scope for a given PACTA sector should be marked `TRUE` in the `borderline` column. Any sector code that is clearly not in scope should be marked `FALSE` and receive the sector value `not in scope`. Any sector code that is clearly in scope should be marked `FALSE` for `borderline` and receive the PACTA sector value that it maps to in the `sector` column.
+Please notice that when manually preparing a custom sector classification file, you need to decide for every code of your mapping system which PACTA sector the code maps to. This requires the system to be relatively granular to capture the nuances of the supply chains of the companies in your loan book. As discussed above, any sector code that is not unambiguously in scope or out of scope for a given PACTA sector should be marked `TRUE` in the `borderline` column. Any sector that code that is clearly not in scope should be marked `FALSE` and receive the sector value `not in scope`. Any sector code that is clearly in scope should be marked `FALSE` for `borderline` and receive the PACTA sector value that it maps to in the `sector` column.
```

Comment:
I think this "that" should not be here

Reply:
correct! sorry the whole text reshuffling is getting a bit confusing